skynet/modules/stt/streaming_whisper/connection_manager.py

```diff
@@ -14,9 +15,14 @@
 
 class ConnectionManager:
     connections: dict[str, MeetingConnection]
+    flush_after_ms: int
 
     def __init__(self):
         self.connections: dict[str, MeetingConnection] = {}
+        self.flush_after_ms = 2000
```

Comment:
Make this a constant in all caps outside of the class please, since we are not changing the value withint the class anyway.